build.gradle

```diff
@@ -67,7 +15,7 @@ buildscript {
 
     dependencies {
         classpath 'com.android.tools.build:gradle:7.3.1'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}"
+        classpath libs.kotlin.gradle.plugin
         classpath "com.google.devtools.ksp:com.google.devtools.ksp.gradle.plugin:1.7.22-1.0.8"
         classpath "org.jlleitschuh.gradle:ktlint-gradle:11.0.0"
```

Comment:
Use `alias` to apply plugins in `plugins`, and move these declarations into catalog.

Reply:
I currently working on the plugins

Reply:
With the plugins I get no further. I also do not know if there is really an added value, because nowhere plugins (apply plugin) with a version number is.

Reply:
You can ref https://github.com/chrisbanes/tivi

Reply:
I hope, I have it now.

Reply:
You should use them in all modules, like https://github.com/chrisbanes/tivi/blob/505646afa4c160443add08d60f0695f724bafa3e/app/build.gradle.kts#L18-L23